doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -7,64 +7,177 @@ In this chapter, we introduce advanced commands to modify the way Coq
 parses and prints objects, i.e. the translations between the concrete
 and internal representations of terms and commands.
 
-The main commands to provide custom symbolic notations for terms are
-:cmd:`Notation` and :cmd:`Infix`; they will be described in the
-:ref:`next section <Notations>`. There is also a
-variant of :cmd:`Notation` which does not modify the parser; this provides a
-form of :ref:`abbreviation <Abbreviations>`. It is
-sometimes expected that the same symbolic notation has different meanings in
-different contexts; to achieve this form of overloading, Coq offers a notion
-of :ref:`notation scopes <Scopes>`.
-The main command to provide custom notations for tactics is :cmd:`Tactic Notation`.
+When inputing a term to Coq, it goes through several successive steps.
```

Comment:
Better to make a more specific claim here than to punt at the end with "elaborating and type checking come later but are out of scope for the current section."  As a reader, I would be pretty disappointed.

Reply:
Tried to improve that, let me know if you have a better solution.

Reply:
Sorry, I had to do a rebase to introduce the "moving things around" commits that you, rightfully, required and then forgot to make temporary change commits. In any case, clicking on the "force pushed" link from Github is a good way to get a diff in those cases.